pandas/_libs/hashtable_func_helper.pxi.in

```diff
@@ -158,9 +167,16 @@ cdef duplicated_{{dtype}}(const {{dtype}}_t[:] values, object keep='first'):
         with nogil:
         {{endif}}
             for i in range(n):
```

Comment:
Would be nice if we could template `for i in ...` based on `{{ if keep == "first"/"last" }}` but could be a follow up.

Reply:
Oh good idea, would do as a follow up to keep diff reviewable.